tests/phpunit/integration/api/image/beans-image-editor/createEditedImage.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Tests for get_image_info() method of _Beans_Image_Editor.
```

Comment:
File name is createEditedImage.php but class name and tested function is get_image_info()
Similar for Unit tests.